.github/issue_template.md

```diff
@@ -1,11 +1,12 @@
 # If your issue looks like a hardware fault or a configuration problem please don't raise an issue here.
 
-## Please consider using other user support options such as asking the manufacturer of the hardware you are using, RCGroups: https://rcgroups.com/forums/showthread.php?t=2464844, or Slack (registration at https://slack.betaflight.com/) or other user support forums & groups (e.g. facebook).
+## Please consider using other user support options such as asking the manufacturer of the hardware you are using, RCGroups: https://rcgroups.com/forums/showthread.php?t=2464844, or Slack (registration at https://slack.betaflight.com/) or other user support forums & groups (e.g. Facebook).
 
 ## If you believe there is an issue with the firmware itself please follow these steps:
 1. Describe your problem;
 2. Include ways to reproduce the problem;
-3. Provide information about your flightcontroller and other components including how they are connected/wired;
-4. Add the used configuration and firmware version;
-5. Create a `diff` and post it here in a code block. Put ``` (three backticks) at the start and end of the `diff` block (instructions  on how to do a diff: https://oscarliang.com/use-diff-not-dump-betaflight/); and
-6. Remove this Text :).
+3. Provide as much information as possible about your hardware and software, including:
+- what hardware / software you are using;
+- the version of all the software used;
+- how things are connected / wired up.
+4. Remove this Text :).
```

Comment:
Are we intending to remove the instructions to supply a `diff`?

Reply:
I tried to standardise the text and make it non-firmware-specific so that it could be added and then updated in all repositories synchronously.
As part of that the `diff` instructions were dropped, but I guess it can't hurt to have one extra line in here.